packages/order-utils/CHANGELOG.json

```diff
@@ -1,4 +1,16 @@
 [
+    {
+        "version": "0.0.5",
+        "changes": [
+            {
+                "note": "Add formatters package for converting signedOrder to orderAddresses & orderValues",
```

Comment:
That is odd. You have an object with the same keys.
You should format it as a list of objects.
Also add a PR key to each one